### PR TITLE
Issue #38 Add links to download config/SEB app when plugin prevents access

### PR DESCRIPTION
**Pull request by andrewmadden**
Added as HTML inside error string. 

Haven't added any styling. HTML may need to be adjusted to better reflect Moodle look.

### CODE REVIEW

**Comment by andrewmadden** [comment]
~~Forgot to update unit tests. Doing now.~~
Done.

**Comment by dmitriim** [comment]
Seems like travis is not happy.

**Comment by dmitriim** [comment]
@andrewmadden can you please provide a screenshot of the screen?

File: lang/en/quizaccess_seb.php [hunk 1]
```diff
@@ -90,5 +90,7 @@
 // Capabilities.
 $string['seb:bypassseb'] = 'Bypass the requirement to view quiz in Safe Exam Browser.';
 
-// Quiz access errors.
+// Quiz access information.
 $string['invalidkeys'] = "The config key or browser exam keys could not be validated. Please ensure you are using the Safe Exam Browser with correct configuration file.";
+$string['sebrequired'] = "This quiz has been configured so that students may only attempt it using the Safe Exam Browser.";
+$string['downloadbutton'] = '<a class="btn btn-secondary mx-1" role="button" href="{$a->link}">{$a->text}</a>';
```
Comment (dmitriim) [inline lang/en/quizaccess_seb.php]: Not sure if this is a good idea to keep HTML code here. Let's move it to template? and move strings like 'text' => 'Launch Safe Exam Browser' to lang strings

**Comment by andrewmadden** [comment]
With access.
Screenshot from 2019-12-20 14-38-51

Without access.
Screenshot from 2019-12-20 14-45-17

File: rule.php [hunk 2]
```diff
@@ -343,4 +375,14 @@ private static function filter_plugin_settings(stdClass $settings) {
         $settings = self::filter_by_prefix($settings);
         return self::strip_all_prefixes($settings);
     }
+
+    /**
+     * Check if Moodle site is using HTTPS.
+     *
+     * @return bool True if HTTPS is detected.
+     */
+    private function is_site_secure() {
```
Comment (dmitriim) [inline rule.php]: is there API for that?

**Pull request merged**